gpu_bdb/build/lib/bdb_tools/__init__.py

```diff
@@ -0,0 +1 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
```

Comment:
Add all files within `build` and it's subfolders to gitignore and remove all the installed lib files.